### PR TITLE
Upgrade guide from v0.2 to v0.3

### PR DESCRIPTION
**Pull request by pedroslopez**
To be merged on release of v0.3

### CODE REVIEW

File: pages/docs/support/upgrading-grouparoo/v02-v03.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: "Upgrading Grouparoo from v0.2 to v0.3"
+date: "2021-04-16"
+pullQuote: "v0.3 is an exciting release with some breaking changes! Learn how to upgrade to v0.3."
+---
+
+Hello Grouparoo Community! v0.3 introduces the core concept of "Sync Modes" and lays the groundwork for adding sync modes across all current and future destinations. We only expect these breaking changes to affect a subset of our users.
```
Comment (evantahler) [inline pages/docs/support/upgrading-grouparoo/v02-v03.mdx]: [syncModes] - link to the blog post so folks can learn more about what they are

Comment (evantahler) [reply pages/docs/support/upgrading-grouparoo/v02-v03.mdx]: > We only expect these breaking changes to affect a subset of our users, specifically those who are configuring Grouparoo via code.

File: pages/docs/support/upgrading-grouparoo/v02-v03.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## Migration Guide
+
+**Upgrading Grouparoo**
+
+1. Update Grouparoo to the latest version through the usual steps by updating all the package version numbers to the latest version. More information is documented in [Installing updates](/docs/support/upgrading-grouparoo#installing-updates).
```
Comment (evantahler) [inline pages/docs/support/upgrading-grouparoo/v02-v03.mdx:13]: Added some enhancements to that page via https://github.com/grouparoo/www.grouparoo.com/pull/381

File: pages/docs/support/upgrading-grouparoo/v02-v03.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+2. Depending on how you've been configuring destinations, some extra steps may be needed:
+   - If using the **Enterprise UI**, no additional steps are required. We have provided migrations for affected plugins that will be automatically applied the next time Grouparoo starts up.
+   - If using **Code Config**, you may need to specify which `syncMode` your destinations will be operating in. For more information, keep reading the following sections.
```
Comment (evantahler) [inline pages/docs/support/upgrading-grouparoo/v02-v03.mdx]: > For more information, keep reading the following sections. 

is a little odd.  How about

> More information can be found below.

File: pages/docs/support/upgrading-grouparoo/v02-v03.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+Hello Grouparoo Community! v0.3 introduces the core concept of "Sync Modes" and lays the groundwork for adding sync modes across all current and future destinations. We only expect these breaking changes to affect a subset of our users.
+
+## Migration Guide
+
+**Upgrading Grouparoo**
+
+1. Update Grouparoo to the latest version through the usual steps by updating all the package version numbers to the latest version. More information is documented in [Installing updates](/docs/support/upgrading-grouparoo#installing-updates).
+2. Depending on how you've been configuring destinations, some extra steps may be needed:
+   - If using the **Enterprise UI**, no additional steps are required. We have provided migrations for affected plugins that will be automatically applied the next time Grouparoo starts up.
```
Comment (evantahler) [inline pages/docs/support/upgrading-grouparoo/v02-v03.mdx]: > You will see a new `syncMode` option for each Destination you already have clarifying the behavior in use.  As we continue to update our plugins, you may be able to change the syncMode used.

File: pages/docs/support/upgrading-grouparoo/v02-v03.mdx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  * parent (required) - The id of the pipedrive App to use for this Destination, e.g: `--parent data_warehouse`
+
+Optional Options:
+  * syncMode - How should profiles sync to the destination? e.g. `--sync-mode additive`. Options: sync, additive, enrich (default: "sync")
+```
+
+The last line of the previous example shows available options and default sync mode for the Pipedrive destination.
+
+### Migrating plugins that previously supported sync modes
+
```
Comment (evantahler) [inline pages/docs/support/upgrading-grouparoo/v02-v03.mdx:75]: ... this is still just for those using code config, yes?

**Review by evantahler — COMMENTED** [review]
This is great and really thorough!  Just some minor notes below

File: pages/docs/support/upgrading-grouparoo/v02-v03.mdx (hunk 5 above)
Comment (pedroslopez) [reply pages/docs/support/upgrading-grouparoo/v02-v03.mdx:75]: yes! Changed the title to an h4 but let me know if it's still unclear

Comment (evantahler) [reply pages/docs/support/upgrading-grouparoo/v02-v03.mdx:75]: I think this covers it!

> **Note that these steps are automatically performed for you if you have been using the UI to configure your destinations.**

**Review by evantahler — APPROVED** [review]

**Pull request merged**